File: Modul-2/C/PRAK202-2310817320007-ZahraNabila.c
#include <stdio.h>

int main()
{
    float NilaiPertama, NilaiKedua, Jumlah;

    printf ("Masukkan Nilai Pertama  : ");
    scanf ("%f", &NilaiPertama);

    printf ("Masukkan Nilai Kedua    : ");
    scanf ("%f", &NilaiKedua);

    Jumlah = NilaiPertama + NilaiKedua;

    printf("Hasil dari penjumlahan nilai pertama \"%g\" dan nilai kedua \"%g\" adalah \"%.2f\"\n", NilaiPertama, NilaiKedua, Jumlah);

    return 0;
}

File: Modul-2/Python/PRAK202-2310817320007-ZahraNabila.py
nilaiPertama = float(input("Masukkan Nilai Pertama  : "))
nilaiKedua   = float(input("Masukkan Nilai Kedua    : "))

jumlah = nilaiPertama + nilaiKedua

nilaiPertama = ( f"{int(nilaiPertama)}" if nilaiPertama.is_integer() else f"{nilaiPertama:.2f}")

print(f'Hasil dari penjumlahan nilai pertama "{nilaiPertama}" dan nilai kedua "{nilaiKedua}" adalah "{jumlah :.2f}"')
